src/gui/widgets/text_box_base.hpp

```diff
@@ -70,6 +70,26 @@ class text_box_base : public styled_widget
 		return text_.get_length();
 	}
 
+	std::vector<std::string> get_lines()
+	{
+		return text_.get_lines();
+	}
+
+	unsigned get_lines_count() const
+	{
+		return text_.get_lines_count();
+	}
+
+	int get_byte_offset(const unsigned column) const
+	{
+		return text_.get_byte_offset(column);
+	}
+
+	void set_highlight_area(const unsigned start_offset, const unsigned end_offset, const color_t& color)
```

Comment:
Please document whether the offsets are in bytes or columns.

Reply:
Added.